frontend/src/modules/containers/actions.js

```diff
@@ -53,11 +53,12 @@ export const list = (options) => async (dispatch, getState) => {
     ));
 };
 
-export const listTable = ({ offset = 0, limit = DEFAULT_PAGINATION_LIMIT } = {}, abort) => async (dispatch, getState) => {
+export const listTable = ({ offset = 0 } = {}, abort) => async (dispatch, getState) => {
```

Comment:
What I meant is not removing the default pagination limit from the params in the actions. 
It's more that in the store there are 2 places where the information is stored (pagination pageSize AND each of the entities page limit), it could be only stored in the pageSize from the pagination state.